packages/sample-app/src/sample_pinecone.ts

```diff
@@ -0,0 +1,92 @@
+import * as traceloop from "@traceloop/node-server-sdk";
+import { Pinecone } from "@pinecone-database/pinecone";
+import * as pc_module from "@pinecone-database/pinecone";
+
+traceloop.initialize({
+  appName: "sample_pinecone",
+  apiKey: process.env.TRACELOOP_API_KEY,
+  disableBatch: true,
+  instrumentModules: {
+    pinecone: pc_module
+  }
```

Comment:
Without this, I'm not getting pinecone traces, but the other samples do not manually include instrument modules. What am I missing here?